js/main.js

```diff
@@ -1,29 +1,35 @@
 $(document).ready(function () {
-  // Show popup notice
-  $('.exception').popup({ position: 'right center', hoverable: true, title: 'Exceptions & Restrictions' });
+  // Make category buttons square
+  $('.box').height($('.box').width());
+
+  // Show region notice
+  if (window.localStorage.getItem('region-notice') !== 'hidden') $('#region-notice').collapse('show');
 
   // Register service worker
   if ('serviceWorker' in navigator) navigator.serviceWorker.register('/service-worker.js');
 
   // Show category of query
   const query = window.location.hash;
-  if (query && query.indexOf('#') > -1) {
-    // Remove all tables before showing the correct one
-    $('.collapse').collapse('hide');
-    showCategory(query.substring(1));
-  }
+  if (query && query.indexOf('#') > -1) showCategory(query.substring(1));
+});
+
+$(window).on('hashchange', function () {
+  const query = window.location.hash;
+  if (query && query.indexOf('#') > -1) showCategory(query.substring(1));
 });
 
+$('.exception').popup({position: 'right center', hoverable: true, title: 'Exceptions & Restrictions'});
+
 // On category click
-$('.cat').click(function () {
-  let query = window.location.hash;
+$('.category-btn').click(function () {
+  let query = window.location.hash.substring(1);
 
   // Collapse all other tables
   $('.collapse').collapse('hide');
```

Comment:
A consequence of this line is that the region notice disappears when a category button is clicked. Do we want to continue showing it until the user has clicked the cross button, or is this behaviour fine (as it will still appear the next time the user visits the site)?